docs/runtimes/create-and-use-functions.ipynb

```diff
@@ -0,0 +1,550 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8d3e3a9c",
+   "metadata": {},
+   "source": [
+    "# Create and Use Functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bfe1aea0",
+   "metadata": {},
+   "source": [
+    "Functions are the basic building blocks of MLRun - they are essentially a Python object that knows how to run locally or on a Kubernetes cluster. This section will cover how to create and customize an MLRun Function as well as common parameters across all functions."
```

Comment:
"Functions are the basic building blocks of MLRun - they are essentially a Python object that knows how to run locally or on a Kubernetes cluster. This section covers how to create and customize an MLRun Function, as well as common parameters across all functions."